src/pow.cpp

```diff
@@ -91,7 +165,7 @@ unsigned int CalculateNextWorkRequired(const CBlockIndex* pindexLast, int64_t nF
          return bnPowLimit.GetCompact();
     }
     // MVF-BU end
-    LogPrintf("  nActualTimespan = %d  before bounds\n", nActualTimespan);
+    LogPrintf("  mvf: nActualTimespan = %d  before bounds\n", nActualTimespan);
```

Comment:
please capitalize the 'mvf' for consistency - makes it easier for me to grep the log outputs